evap/staff/views.py

```diff
@@ -1893,6 +1898,16 @@ def text_answer_warnings_index(request):
 
 @manager_required
 def user_index(request):
```

Comment:
It would be nice if you could add a small test for this view that loads the page and submits the user selection form, asserting that the redirect happens as expected.